homeassistant/components/light/flux_led.py

```diff
@@ -227,10 +227,12 @@ def turn_on(self, **kwargs):
                               random.randint(0, 255))
         elif effect in EFFECT_MAP:
             self._bulb.setPresetPattern(EFFECT_MAP[effect], 50)
+        self._bulb.update_state(retry=2)
 
     def turn_off(self, **kwargs):
         """Turn the specified or all lights off."""
         self._bulb.turnOff()
+        self._bulb.update_state(retry=2)
```

Comment:
Is there a reason this isn't just calling self.update() here, which is effectively the same call except with error recover.

Reply:
I don't have a better answer than the implementation was just that suggested by @Danielhiversen in the issue report.

Reply:
I think it's probably safer to use self.update() here because it keeps all the error handling as well. So how about update for that, and we can merge.

Reply:
Ok, given that this would have already exploded if there wasn't a connection, this is probably fine.